package.json

```diff
@@ -4,57 +4,66 @@
   "description": "A toolkit for conducting machine learning trials against confidential data",
   "license": "MIT",
   "scripts": {
-    "test": "mocha build/test/unit",
-    "pretest-functional": "npm run compile",
-    "test-functional": "mocha build/test/functional --config test/functional/.mocharc.yml",
+    "test": "mocha",
+    "pretest:functional": "npm run compile",
+    "test:functional": "mocha --config test/functional/.mocharc.yml",
     "check": "gts check",
-    "clean": "gts clean",
+    "clean": "gts clean && rimraf ./dist",
     "compile": "tsc -p .",
+    "dct": "node build/src/cli/dct.js",
+    "dct:dev": "npm run compile && node -r dotenv/config build/src/cli/dct.js",
     "fix": "gts fix",
     "prepare": "npm run compile",
     "pretest": "npm run compile",
     "posttest": "npm run check",
-    "worker": "node build/src/worker/app.js"
+    "laboratory": "node build/src/laboratory/server/main.js",
+    "laboratory:dev": "npm run compile && node -r dotenv/config build/src/laboratory/server/main.js",
+    "laboratory:package:appservice": "npm run compile && mkdir -p dist && cp -r --parents package*.json build/src dist && cd dist && npm install --ignore-scripts --production && zip -r $npm_package_name.zip package.json package-lock.json build/src node_modules",
+    "worker": "node build/src/worker/app.js",
+    "worker:dev": "npm run compile && node -r dotenv/config build/src/worker/app.js"
```

Comment:
So the idea here is that the codebase won't know about dotenv? Is the intention to use the .env file when doing dev work, and let Azure set the environment variables in the cloud?

Reply:
Exactly